src/js/control_plugins/starRating.js

```diff
@@ -36,15 +36,32 @@ window.fbControls.push(function(controlClass) {
      * @return {Object} DOM Element to be injected into the form.
      */
     build() {
-      return this.markup('span', null, {id: this.config.name});
+      this.inputId = this.config.name;
+      let defaultValue = this.config.value || 3.6;
+      this.input = this.markup('input', null, {
+        name: this.inputId,
+        id: this.inputId,
+        value: defaultValue,
```

Comment:
`defaultValue` isn't used anywhere else so I think this could just be
```suggestion
        value: this.config.value || 3.6,
```